pkg/logger/logger.go

```diff
@@ -148,6 +148,31 @@ func (rl *ReceptorLogger) Debug(format string, v ...interface{}) {
 	rl.Log(DebugLevel, format, v...)
 }
 
+// Debug payload data.
+func (rl *ReceptorLogger) DebugPayload(payloadDebug int, payload string, workUnitID string, connectionType string) {
+	switch payloadDebug {
+	case 3:
```

Comment:
Potentially 3 traces messages per payload seems nosey to me, Can we use these cases to "build" a log message instead and just log it once please?